Makefile

```diff
@@ -3,6 +3,7 @@
 # - emu: cpu emulator-based native build
 # - rosetta: rosetta-based native x86 build
 # - unicorn: cpu emulator-based native build
+# - linux: linux based x86 build
```

Comment:
I think this isn't in this file (?)

Reply:
You are correct, that was a residual that I left in when I added a separate sdl-linux.sh. I opted to altering the `brew.sh` script instead.